partiql/tests/queries.rs

```diff
@@ -0,0 +1,30 @@
+use crate::common::{eval_query_with_catalog, TestError};
+use assert_matches::assert_matches;
+use partiql_catalog::catalog::PartiqlCatalog;
+use partiql_catalog::extension::Extension;
+use partiql_eval::eval::Evaluated;
+use partiql_eval::plan::EvaluationMode;
+use partiql_extension_value_functions::PartiqlValueFnExtension;
+use partiql_value::Value;
+
+mod common;
+
+#[track_caller]
+#[inline]
+pub fn eval(statement: &str, mode: EvaluationMode) -> Result<Evaluated, TestError<'_>> {
+    let mut catalog = PartiqlCatalog::default();
+    let ext = PartiqlValueFnExtension::default();
+    ext.load(&mut catalog)?;
+
+    eval_query_with_catalog(statement, &catalog, mode)
+}
+
+#[test]
+fn select_star_unpivot() {
+    let query = "SELECT * FROM UNPIVOT {'amzn': 840.05, 'tdc': 31.06} AS price AT sym";
```

Comment:
could be a good test we port to conformance tests

Reply:
https://github.com/partiql/partiql-tests/issues/134